vendor/k8s.io/kube-openapi/pkg/util/trie.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Is this now duplicated code? I did not see you deleting the copy in staging/src/k8s.io/apiserver/pkg/util/trie/trie.go